Change "authentication token" to "API token" in InfluxDB 2 docs (#2929)
influxdata/docs-v2: content/influxdb/v2.0/security/tokens/_index.md

weight: 103
---

> InfluxDB **API tokens** ensure secure interaction between users and data.

Reviewer: ```suggestion
InfluxDB **API tokens** ensure secure interaction between InfluxDB and external tools such as clients or applications.
```

> A token belongs to an organization and identifies InfluxDB permissions within the organization.

Reviewer: ```suggestion
An API token belongs to an organization and identifies InfluxDB permissions within the organization.
```

Learn how to create, view, update, or delete an API token.

## API token types

> - [Operator token](#operator-token)

Reviewer: ```suggestion
- [Operator API token](#operator-token)
```

> - [All-Access token](#all-access-token)

Reviewer: ```suggestion
- [All-Access API token](#all-access-token)
```
